samcli/lib/package/artifact_exporter.py

```diff
@@ -62,12 +69,7 @@ def do_export(self, resource_id, resource_dict, parent_dir):
 
         template_path = resource_dict.get(self.PROPERTY_NAME, None)
 
-        if (
-            template_path is None
-            or is_s3_url(template_path)
-            or template_path.startswith(self.uploader.s3.meta.endpoint_url)
```

Comment:
explain: `self.uploader.s3.meta.endpoint_url` looks like `https://s3.us-west-2.amazonaws.com`. I think originally @sriram-mv 's intention is to make sure the url is in the same region. (@sriram-mv correct me if wrong) I think we don't need to handle it here because if the url lacks region info (like `s3://bucket-name/key`), we let it pass through. So we can have a consistent behavior here.